frameworks/digital-outcomes-and-specialists/questions/brief-responses/essentialRequirements.yml

```diff
@@ -1,4 +1,5 @@
 name: Give evidence of the essential skills and experience
+short_name: Essential skills and experience
```

Comment:
So I think we are able to change the name. I think we were quite new to the content loader when we started so assumed you needed both a name and question (and therefore set them to be the same thing).

Therefore I think we are able to have `name` be `Essential skills and experience` and not need the new `short_name` key. Sorry!